docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 pyparsing
 pyperclip
 wcwidth
+colorama
```

Comment:
Thanks for adding this.  We need to keep this up-to-date so that the documentation builds correctly on ReadTheDocs